### DISCUSSION TITLE
Cannot use cmdVel

### DISCUSSION THREAD

**Discussion by adolhongping** [USC-ACTLab/crazyswarm#395] [Q&A]
When i use cmdVel(), I also use rostopic echo /cf2/cmd_vel to monitor the order. It shows that some order has published to the drone, but the drone do not move, even if I have given thrust of 2^16 to the drone. I also tried to  firstly send  few messages such as 0 0 0, but  the drone did not respond.

>

**Comment by Wallat1** [comment]
Hi.

@adolhongping  Have you solved the problem?.

Regards.

**Comment by powehi2020** [comment]
Hi.

@adolhongping Have you solved the problem?.
